src/usr/local/www/widgets/widgets/openvpn.widget.php

```diff
@@ -46,7 +46,13 @@ function printPanel($widgetkey) {
 			$got_ovpn_server = true;
 
 		$opstring .= "<div class=\"widget panel panel-default\">";
-		$opstring .=	"<div class=\"panel-heading\"><h2 class=\"panel-title\">" . htmlspecialchars($server['name']) . "</h2></div>";
+		$opstring .= "<div class=\"panel-heading\"><h2 class=\"panel-title\">" .
+		    htmlspecialchars($server['name']);
+		if ($server['conns'][0]['common_name'] != '[error]') {
+			$opstring .= gettext('Client Connections') . ": " . 
```

Comment:
This needs a space concatenated to the string before the gettext call (e.g. `.= ' ' . gettext(...`), right now it's run together so it shows up like `Remote Internet Access UDP4:1194Client Connections: 1`.